docs/modules/ROOT/pages/config-safe.adoc

```diff
@@ -1,37 +1,29 @@
 = Securing Cluster Configuration Using the Config Safe
 
-:description: This guide shows you how to store cluster configuration in the encrypted storage.
+:description: This guide shows you how to store cluster configuration in encrypted storage.
 
 {description}
 
 == Introduction
 
-Cluster configuration is one of the central ideas of Hazelcast CLC.
-CLC provides a convenient way of connecting to different clusters by their names.
-Although CLC provides several ways of creating new cluster configurations, it is also very easy to add and manipulate them.
-They are just YAML files in `$CLC_HOME/configs` directory after all!
+By default, cluster configuration is stored as plain text YAML files in the `$CLC_HOME/configs` directory. This provides convenient access, but sensitive data such as user credentials and SSL key passwords should be stored more securely. CLC's *Config Safe* provides encrypted storage for cluster configuration that includes sensitive data.
 
-This way of storing configuration is very common among CLI tools, and it is also convenient to the user.
-But some cluster configurations may include sensitive data, such as user credentials and SSL key passwords.
-In those cases, it may be preferable to let go of a bit of convenience for more secure storage of the cluster configuration.
+The Config Safe is an encrypted directory at `$CLC_HOME/configsafe-v1`. It is secured with AES-128 (16 bytes binary, or 32 bytes string key). Configuration in the Config Safe is separate from the default configuration, so it is possible to have different configuration objects with the same name in each directory. You can only access the Config Safe by adding the `--safe` flag to CLC commands and providing the encryption key.
```

Comment:
The meaning of the default configuration is a bit different in CLC.
The user is able to name a configuration `default` and in that case it would be used if a configuration name is not specified with `-c/--config` flags.

Would it make sense to have something along the lines of "regular" configuration store instead of default configuration?

> so it is possible to have different configuration objects with the same name in each directory

IMO it would be better not to mention "directory" here, since that's an implementation detail.
Would it make sense to change it to something like:

> Configuration in the Config Safe is separate from the regular configuration store, so it is possible to have different configuration objects with the same name between them.